conf/ccga.config

```diff
@@ -0,0 +1,72 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Nextflow config file with environment modules for RZCluster in Kiel
+ * -------------------------------------------------
+ */
+
+executor {
+  queueSize=100
+}
+
+process {
+
+  // Global process config
+  executor = 'slurm'
+  queue = 'ikmb_a'
+
+  clusterOptions = { "--qos=ikmb_a" }
+  
+  // Load bioinfo-tools module before we start
+  beforeScript = 'module load IKMB'
+
+  // Environment modules and resource requirements
+  $makeSTARindex.module = ['STAR/2.5.2b']
```

Comment:
Ouch - no singularity / conda?

Reply:
nope, requested conda to be installed, but god knows how long that might take

Reply:
Sad times.... 😢